lib/tasks/hydranorth.rake

```diff
@@ -16,9 +16,9 @@ namespace :hydranorth do
 
   desc 'Run rubocop and specs for CI'
   task ci: :environment do
-    Rake::Task['hydranorth:rubocop'].invoke
     with_test_server do
       Rake::Task['hydranorth:rspec'].invoke
     end
+    Rake::Task['hydranorth:rubocop'].invoke
```

Comment:
Reordering this so rspec runs before rubocop. So a tiny code style change doesn't prevent specs from running when running the command. For example, if you add like a `binding.pry` into a spec file.